api/errors/errors.go

```diff
@@ -1,47 +1,57 @@
 package errors
 
-import "net/http"
+// NotFoundError signals that the requested object doesn't exist
+type NotFoundError interface {
+	error
```

Comment:
Secondary interface overloading is not required. Just define the single `NotFound` method.